Warning: OMP_NUM_THREADS=16 is greater than available PU's
Warning: OMP_NUM_THREADS=16 is greater than available PU's
Traceback (most recent call last):
  File "pytorch_experiment.py", line 388, in <module>
    main_experiment()
  File "pytorch_experiment.py", line 103, in main_experiment
    dist.init_process_group(backend="nccl")
  File "/gpfs/alpine/gen011/world-shared/native-build/anaconda3/envs/WML161/lib/python3.6/site-packages/torch/distributed/distributed_c10d.py", line 432, in init_process_group
    store, rank, world_size = next(rendezvous(url))
  File "/gpfs/alpine/gen011/world-shared/native-build/anaconda3/envs/WML161/lib/python3.6/site-packages/torch/distributed/rendezvous.py", line 119, in _env_rendezvous_handler
    raise _env_error("RANK")
ValueError: Error initializing torch.distributed using env:// rendezvous: environment variable RANK expected, but not set

------------------------------------------------------------
Sender: LSF System <lsfadmin@batch2>
Subject: Job 537587: <data_challenge_modeling> in cluster <summit> Done

Job <data_challenge_modeling> was submitted from host <login1> by user <shutoaraki> in cluster <summit> at Fri Aug  2 09:58:55 2019
Job was executed on host(s) <1*batch2>, in queue <batch>, as user <shutoaraki> in cluster <summit> at Fri Aug  2 09:59:04 2019
                            <42*h36n10>
</ccs/home/shutoaraki> was used as the home directory.
</ccs/home/shutoaraki/datachallenge2/pytorch> was used as the working directory.
Started at Fri Aug  2 09:59:04 2019
Terminated at Fri Aug  2 10:06:49 2019
Results reported at Fri Aug  2 10:06:49 2019

The output (if any) is above this job summary.

